README.md

```diff
@@ -28,6 +28,14 @@
    * [React Documentation](https://reactjs.org/docs/getting-started.html)
    * [React Tutorial](https://reactjs.org/tutorial/tutorial.html)
 
+* Vue
+   * [Vue Documentation](https://vuejs.org/v2/guide/)
+   * [Vee Validate](https://baianat.github.io/vee-validate/) - Client-side Validation
```

Comment:
Phew. Thank god _vee_ is in the name of the library

Reply:
I had to double-check that too. :+1:

Reply:
I saw this and panicked! 😄

Reply:
What's Vue?